tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -173,7 +174,8 @@ def prepare_ir(argv: argparse.Namespace):
         moc_front_end.add_extension(TelemetryExtension("ovc", t.send_event, t.send_error, t.send_stack_trace))
         if any_extensions_used(argv):
             for extension in argv.extension:
-                moc_front_end.add_extension(extension)
+                if not isinstance(extension, ModuleExtension):
```

Comment:
BTW, why do we need this check? Frontend should filter out unknown extensions.

Reply:
It won't be able to wrap in OVAny